build.sbt

```diff
@@ -9,7 +9,7 @@ organizationName in ThisBuild := "Jeff May"
 
 semVerLimit in ThisBuild := "2.0.999"
 
-scalaVersion in ThisBuild := "2.11.8"
+scalaVersion in ThisBuild := "2.11.11"
 crossScalaVersions in ThisBuild := Seq("2.11.8", "2.10.6")
```

Comment:
Hmm, I think Scala 2.11 doesn't necessarily imply that the use has Java 8...

Maybe I should separate the java.time stuff? Also, with Java 9, it might be something you have to explicitly request as a module.

Reply:
Also, I might want to remove Scala 2.10, since nobody should still be using that...